constrained-decls.html

```diff
@@ -298,6 +298,11 @@ <h3>General rules</h3>
 
 template &lt;Constructible&lt;int&gt; auto N&gt; void f();</code></pre></blockquote>
 
+<p>
+  Variadic concept identifiers constrain the types deduced for the pack when used to declare a pack (thus adding <code>...</code> to <code>Constraint auto</code> can be a more loaded operation than usual):
+</p>
+<blockquote><pre><code>void f(Homogeneous auto&amp;&amp;... x); // Homogeneous&lt;decltype(x)...&gt;</code></pre></blockquote>
```

Comment:
Sigh; it seems wrong to me that such a declaration declares a function applying a constraint to a pack, rather
than declaring a function with a pack of constrained types with the constraint applied to each element of the pack. Is that what the TS says?

Reply:
That is what the TS says for a variadic concept, yes. It does the other thing for non-variadic concepts.

For the case above, if `Homogeneous` is a variadic concept, then the TS says that we go through the machinery for a <i>constrained-parameter</i> that is a parameter pack. The WD already has the wording for that machinery already.

The weird thing to me is allowing a variadic concept as a placeholder in other-than-pack contexts.